controllers/activemqartemis_controller.go

```diff
@@ -310,7 +327,33 @@ func (r *ActiveMQArtemisReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	builder := ctrl.NewControllerManagedBy(mgr).
 		For(&brokerv1beta1.ActiveMQArtemis{}).
 		Owns(&appsv1.StatefulSet{}).
-		Owns(&corev1.Pod{})
+		Owns(&corev1.Pod{}).
+		Watches(
```

Comment:
I think we can avoid the watch by requiring an entry "xx-logging-config" in extra mounts. This makes it an explicit attribute of the CR.

Reply:
make sense. It'll be just like how xx-jaas-config works.